openmls/src/messages/proposals.rs

```diff
@@ -100,6 +117,7 @@ pub enum Proposal {
     PreSharedKey(PreSharedKeyProposal),
     ReInit(ReInitProposal),
     ExternalInit(ExternalInitProposal),
+    // TODO: `AppAck` is not in draft-ietf-mls-protocol-16.
```

Comment:
When creating a TODO we adopted the rule that we reference a ticket. If no ticket exists for the TODO, we create a new one.